### ISSUE TITLE
CMake update fails

### ISSUE THREAD

**Issue by Barben360** [microsoft/vscode-cmake-tools#622]
### Brief Issue Summary
CMake update crashes when accepting the update from VS Code pop-up.

### Expected:

CMake updates to new version

### Apparent Behavior:

1. CMake update message pops up
2. I accept
3. It closes after one second

### Developer Tools Log

```
mainThreadExtensionService.ts:42 [vector-of-bool.cmake-tools]Non-200 response when downloading new CMake installer
e.$onExtensionRuntimeError @ mainThreadExtensionService.ts:42
mainThreadExtensionService.ts:43 Error: Non-200 response when downloading new CMake installer
	at ClientRequest.tmp.file.https.get.res (/home/bbarrois/.vscode/extensions/vector-of-bool.cmake-tools-1.1.3/out/src/cm-upgrade.js:35:32)
	at Object.onceWrapper (events.js:315:30)
	at emitOne (events.js:116:13)
	at ClientRequest.emit (events.js:211:7)
	at HTTPParser.parserOnIncomingClient (_http_client.js:551:21)
	at HTTPParser.parserOnHeadersComplete (_http_common.js:117:23)
	at TLSSocket.socketOnData (_http_client.js:440:20)
	at emitOne (events.js:116:13)
	at TLSSocket.emit (events.js:211:7)
	at addChunk (_stream_readable.js:263:12)
	at readableAddChunk (_stream_readable.js:250:11)
	at TLSSocket.Readable.push (_stream_readable.js:208:10)
	at TLSWrap.onread (net.js:594:20)
```

### Platform and Versions

- **Operating System**:  Ubuntu 18.04
- **CMake Version**: 3.12.3
- **VSCode Version**: 1.30.1
- **CMake Tools Extension Version**: 1.1.3
- **Compiler/Toolchain**: GCC 7.3

**Comment by Quincunx271** [comment]
Here's what's happening: the github URL it's trying to download from ( https://github.com/Kitware/CMake/releases/download/v3.13.2/cmake-3.13.2-Linux-x86_64.sh ) returns a status of 302, not 200. The html body is:

```html
<html><body>You are being <a href="https://github-production-release-asset-<redacted>.s3.amazonaws.com/<redacted>=attachment%3B%20filename%3Dcmake-3.13.2-Linux-x86_64.sh&amp;response-content-type=application%2Foctet-stream">redirected</a>.</body></html>
```

So we need to follow the redirect and download from there. FWIW, `wget` does follow the redirect, so maybe we can just call something like it in the background rather than implement the logic to handle different status codes ourselves.

**Comment by Quincunx271** [comment]
Actually, this is a duplicate of #614 which was fixed in 7ac8606342f9da5e5b4a649e3d30412069244b5d

**Comment by bobbrow** [comment]
Closing issue marked as duplicate.
